keyboards/fungo/readme.md

```diff
@@ -0,0 +1,23 @@
+# fungo split keyboard (fungo)
+
+![fungo](https://github.com/bojiguard/fungo-split-keyboard/blob/master/picture/full%20cover.jpg)
```

Comment:
please rehost this on imgur or a similar image hosting site.

Reply:
delete image link